duckduckgo/chrome-zeroclickinfo
# include other tracker lists

**Reviewer:**



## Description:


There are a few things going on in here:

1. tracker lists are being pre-processed to produce our internal data structures. You can do this by running grunt. 

2. When the extension starts it will read these pre-processed files. Pre-processing the data speeds up the extension startup time and less likely to have runtime errors.

3. Some clean up of the tracker blocking module

4. Added tests for the tracker blocking module and a readme in /scripts explaining how to add new tracker list importing modules. 

<img width="1021" alt="screen shot 2017-04-21 at 3 08 31 pm" src="https://cloud.githubusercontent.com/assets/1882892/25296233/7b1a4b2a-26a4-11e7-949a-adcd5dbd3e73.png">

## Steps to test this PR:

1. run `grunt`
2. load extension
3. run tests http://[extensionID]/tests/index.html
4. Visit some sites and check that tracker blocking is working

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
